config/fam_metadata_config.yaml

```diff
@@ -4,4 +4,4 @@
 metadata_manager : "radixtree"
  
 # Path where metadata info is stored. By default, it will be placed in /dev/shm. 
-metadata_path: /dev/shm/
+metadata_path: /dev/shm/radixtree
```

Comment:
In line 4 you have metadata_manager : "radixtree" and in line 7 you have metadata_path : /dev/shm/radixtree.

If these two lines need to stay in sync (i.e., if you change metadata_manager : "redis", you want to change metadata_path : /dev/shm/redis, then consider using something like metadata_path : /dev/shm/<metadata_manager> so you only need changes in one place rather than both.